specification/2_DataModel/overview.md

```diff
@@ -0,0 +1,66 @@
+# DataModel
+
+The Data Transmission Format was inspired by a project from the Swedish Defence Materiel Administration.
+
+The specification comprises a SysML Block that represents the template and an associated Parametric Diagram that defines the templates and Platform specific objects instantiated by the MeterReading template.
+
+[Template MeterReading](http://www.plcs.org/plcslib/plcslib/data/contexts/SwedishDefence/templates/MeterReading/template.html)
+
+
+The Swedish Defence Materiel Administration provides a diagram of the data structure used in the system. Because the whole structure is to specific and not needed in this detail, we have reduced it to the following subset that SHOULD be implemented in this system. The following image shows this subset.
```

Comment:
```suggestion
The Swedish Defence Materiel Administration provides a diagram of the data structure used in the system. Since the entire structure is too specific and not needed at this level of accuracy, we have reduced it to the following subset that SHOULD be implemented in this system. The following image shows this subset.
```